binderhub/tests/test_repoproviders.py

```diff
@@ -209,6 +210,34 @@ async def test_dataverse(
     assert spec == resolved_spec
 
 
+@pytest.mark.parametrize(
+    "spec,resolved_spec,resolved_ref,resolved_ref_url,build_slug",
+    [
+        [
+            "https://demo.ckan.org/dataset/sample-dataset-1",
+            "https://demo.ckan.org/dataset/sample-dataset-1",
+            "sample-dataset-1.v",
+            "https://demo.ckan.org/dataset/sample-dataset-1",
```

Comment:
Can you add some additional tests here for `/history/` as well as `?activity_id=` URLs?

Reply:
Thanks, added.